lucille-core/src/main/java/com/kmwllc/lucille/core/OpenAIEmbeddingModel.java

```diff
@@ -0,0 +1,37 @@
+package com.kmwllc.lucille.core;
+
+
+import com.typesafe.config.Config;
+
+public enum OpenAIEmbeddingModel {
```

Comment:
should go in com.kmwllc.lucille.stage.util package

Reply:
resolved in [feeaa04](https://github.com/kmwtechnology/lucille/pull/171/commits/feeaa045fb3aa12e3a184b1bfa7564aafdd21d1d)